lib/live-stream.js

```diff
@@ -31,34 +29,57 @@ function streamData (args, archives, cb) {
       delete archives[link]
       cb()
       dat.listStream.destroy()
-      if (dat.path.indexOf(root) > -1) {
-        rmrf(dat.path, err => {
-          if (err) throw err
+      dat.close(function (err) {
+        if (err) console.error(err)
+        dat.db.close(function (err) {
+          if (err) console.error(err)
+          if (dat.path.indexOf(root) > -1) {
+            rmrf(dat.path, err => {
```

Comment:
Should we remove people's files on their computer now while we're still in testing/beta mode? I think we might get some nasty complaints if people accidentally delete something important when they thought they were just removing it from the list.